zipline/utils/serialization_utils.py

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 from six import BytesIO
-import pickle
+import dill as pickle
```

Comment:
I'll add dill to the front-end as well to keep things consistent

Reply:
It might not be absolutely necessary: the output produced by `zipline run blahblah -o result.pickle` is still pickled by Python's cPickle. 

We only use dill for the internal state persistence.